src/components/display/Table/styles.ts

```diff
@@ -33,52 +33,54 @@ export const TableView = styled.table`
   border-spacing: 0;
   overflow: hidden;
   width: 100%;
-  font-family: "Poppings", sans-serif;
-
-  ${({ theme }) => `
-    background: ${theme.table.backgroundColor};
-    
-    thead {
-      border: 1px solid ${theme.table.thead.borderColor};
-      background: ${theme.table.thead.backgroundColor};
-      
-      th {
-        color: ${theme.table.thead.th.textColor};
-        padding: 12px 22px;
-      }
-    }
-    
-    tbody {
-      td {
-        font-size: 14px;
-        font-style: normal;
-        font-weight: 600;
-        line-height: 20px;
-        letter-spacing: 0px;
-        text-align: center;
-        color: ${theme.table.tbody.textColor};
-        padding: 12px 22px;
-        white-space: nowrap;
-        
-        &.action {
-          display: flex;
-          align-items: center;
-          justify-content: flex-end;
-        }
-      }
+
+  font-family: "Avenir", sans-serif;
```

Comment:
Is it supposed to be "Avenir" or "Avenir-next Redhat" ?

In design I think it was "Avenir-next"

@NataliaAvila-PM

Reply:
Hi! Yes, it should be Avenir Next as primary, if not Red Hat Text, and if not sans-serif
font-family: Avenir Next, Red Hat Text, sans-serif

Reply:
reviewing...